### PR TITLE
Add switch statement

### LINKED ISSUE #23: Fill out `Control Flow / Switch` page
The `/control-flow/switch-statements` page is a stub. Add content.

### PR DESCRIPTION
**Pull request by dj2**
This CL adds content to the `switch-statement` page.

Fixes #23

### CODE REVIEW

File: content/05-control-flow/2-switch-statements.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
 
-# Switch Statements
+Similar to `if` statements, a `switch` statement can be used to branch
+over multiple blocks.
+
+A switch has a condition, the condition must be a [concrete integer
+scalar]({{< ref "/02-types/01-basic-scalars" >}}) type. The `case`
+selectors must have the same type as the condition.
+
+Like the `if` statement the parenthesis around the condition are
```
Comment (dneto0) [inline content/05-control-flow/2-switch-statements.md:19]: nit: Here I think its "parentheses" because you need both (a plural)

Comment (dneto0) [inline content/05-control-flow/2-switch-statements.md:19]: I'd put a comma after the `if`

File: content/05-control-flow/2-switch-statements.md [hunk 2]
```diff
@@ -1,6 +1,52 @@
...
+optional.
+
+A switch can have zero or more `case` blocks.
+
+A `default` block is required. Multiple `default` blocks are not
+permitted
+
+`case` and `default` blocks require brackets.
```
Comment (dneto0) [inline content/05-control-flow/2-switch-statements.md:27]: brackets -> braces

**Review by dneto0 — APPROVED** [review]
Approved with nits.

File: content/05-control-flow/2-switch-statements.md (hunk 1 above)
Comment (dj2) [reply content/05-control-flow/2-switch-statements.md]: done

Comment (dj2) [reply content/05-control-flow/2-switch-statements.md]: Reworded and added comma.

**Pull request merged**